### DISCUSSION TITLE
Text to speech & question about CBZ

### DISCUSSION THREAD

**Discussion by ghost** [koreader/koreader#11656] [Q&A]
1. Text to speech function.
Does KOReader have this function? How to turn it on? Are English and Russian languages available? 
I didn't see the text-to-speech option in the menu...

2. CBR
Can I translate words in pictures (CBZ file)? like in PDF? (I haven't tried it in PDF yet too, but heard about)
I forgot what this function is called.
Please help to install this, if it is possible🙏 

And I'm not programmer, so can you explain all steps for me more in detail.

**Comment by mergen3107** [comment]
No text to speech. 

CBZ: no, these are just images. I would just assign dictionary to a gesture and type in manually

**Reply by dmalinovsky** [reply]
The 2nd function is called OCR and it's supported for the scanned PDFs: https://github.com/koreader/koreader/wiki/Dictionary-support#dictionary-lookups-in-scanned-pages

I guess it shouldn't be too difficult to make it work in the images as well?

**Reply by dmalinovsky** [reply]
As for TTS, see this: https://github.com/koreader/koreader/issues/545

It's unlikely to be supported, unless someone goes to an effort to write a plugin (and it'll be platform specific, as I understand).

**Reply by Frenzie** [reply]
> I guess it shouldn't be too difficult to make it work in the images as well?

It already does. It's just that results are generally worse in comics because the OCR isn't great at hand lettering.
